### ISSUE TITLE
Goals - Sprint 59 Ariba

### ISSUE THREAD

**Issue by manjunathanagaraj** [SAP/fundamental-ngx#5022]
Artur:
Multi Combobox(platform) - PR merge

Yury:
UserActionsMenu -  PR https://github.com/SAP/fundamental-ngx/pull/4502  merge
Value Help Dialog: https://github.com/SAP/fundamental-ngx/issues/4594 - approvals are in place, waiting for merge.

Platon:
Platform Table phase4 - PR merge. 
Table component a11y issues #3845, #3844, #3843, #3841 
Approval Flow feature requirements - based on discussions, may have to come up with tech spec.
(Tree List - after discussing with Oleksii)

Kavya:
Platform Dynamic Page: #4972 and #3744 - PR https://github.com/SAP/fundamental-ngx/pull/4997 merge
Address Component: Tech Spec
Form facets - PR merge (incase not able to merge it by 26th March EoB)

Manu: Rich Text Editor - styles - PR merge - PR https://github.com/SAP/fundamental-styles/pull/2215
Rich Text Editor - Tech Spec - core/platform
https://github.com/SAP/fundamental-ngx/issues/3497 - re-work.

Deepak: Resizable Card Layout - PR merge
Currency Picker - Tech Spec
Money Field - requirements discussion with design team

Oleksii
Moment Date Time updater - PR merge - PR https://github.com/SAP/fundamental-ngx/pull/4592 is still in WIP. 
Tree List - PR merge 

Lok:
Platform page footer - PR merge - PR https://github.com/SAP/fundamental-ngx/pull/4936 raised, review in progress
Defects: 
  -  #4745,  #4781,  #4988 - multi input - PR raised, review in progress
Duration (platform)

Sean:
- Finish writing dialog (core) e2e tests 
- Finish writing calendar (core) e2e tests 
- Write illustrated-message (core) e2e tests
- Write standard list (core)  e2e tests 
- Manual testing for table phase 4 
- Github Actions PoC
- Test/Review PRs

Anton:
- Write inline help test cases
- Write checkbox (core) e2e tests 
- Refactor visual regression test cases 
- Refactor tests with hardcoded colors
-  Write/Update QA wiki docs 
- Test/Review PRs

Nikita (available 5 days):
- Address PR comments 
- Write text (core) manual test cases + e2e tests 
- Write list by line (core) manual test cases + e2e tests 
- Test/Review PRs 

Overall Goals:
-	Complete the PR merges for 7 components
o	                  spill over components : Form Facet, Resizable Card Layout, Page Footer, User Actions Menu, TreeList , Table Phase4
o	Other planned components: Multi Combobox(platform)
-	Tech specs for Rich Text Editor, Currency Picker, Address
-	New app components – Duration, Currency Picker (PR merge is sprint 60)

**Comment by manjunathanagaraj** [comment]
**Final Status:**

Artur:
Multi Combobox(platform) - PR merge - moving to Spring60. PR is not yet raised.

Yury:
UserActionsMenu -  PR https://github.com/SAP/fundamental-ngx/pull/4502  merge- few more changes identified, moving to Sprint 59, new PR 2243 raised for Styles.
Value Help Dialog: https://github.com/SAP/fundamental-ngx/issues/4594 - approvals are in place, waiting for merge. - PR4661 merged.
Fixed broken main branch - PR5062 merged.


Platon:
Was on leave for 3 days.
Platform Table phase4 - PR merge. 
Table component a11y issues #3845, #3844, #3850 - fixed. #3843, #3841 -moving to sprint60
Approval Flow feature requirements - had discussions with Sourcing team, got few more clarifications on features. Will be addressed in sprint 60.
Tree List - could not work on it. Will be taken up later, as the focus now would be on fixing bugs.


Kavya:
Second half of the sprint on leave.
Platform Dynamic Page: #4972 and #3744 - PR https://github.com/SAP/fundamental-ngx/pull/4997 merge - moved to sprint 60
Address Component: Tech Spec - did preliminary analysis, to be taken up later. 
Form facets -  PR4977 merged.

Manu: Rich Text Editor - styles - PR https://github.com/SAP/fundamental-styles/pull/2215 merged.
Rich Text Editor - Tech Spec - core/platform - presentation done, to be presented again after review comments incorporated. To be taken up later.
https://github.com/SAP/fundamental-ngx/issues/3497 - PR merged.

Deepak:
On leave for 2 days.
Refactored Cards to move some of the styles from ngx to Styles - PR 2336 merged. Resizable Card Layout - PR merge - to be moved to sprint60
Currency Picker - Tech Spec - to be taken up later.
Money Field - requirements discussion with design team - to be taken up later.
Card A11Y issues - (#3928 #3929 #4710 #4243 #3863 and #3864) - PR merged https://github.com/SAP/fundamental-ngx/pull/5015 (some of the issues are to be closed by ATL team, moved to sprint 60)

	
Oleksii
Moment Date Time updater - PR merge - Pending since many sprints. PR https://github.com/SAP/fundamental-ngx/pull/4592 is still in WIP. Hope new team member will take up soon and complete this.



Lok:
Platform page footer - Initially styles was included in th platform. Later thr was suggestion to separate it. PR for styles is raised. Once styes PR is merged, platform PR will b factored and merged. 
 
Defects: 
  -  #4745,  #4781,  #4988 - multi input - PR raised, review in progress
Duration (platform)

Sean:
- Calendar e2e tests - PR in review - Dialog e2e tests - PR in review - Standard List e2e tests - PR in review - Github Actions PoC - in progress - Illustrated Message e2e tests - in progress - Manual testing for Table phase 4 - 1st round of testing done, comments left: Completed - Testing Wiki's - Updated visual regression wiki per comments and wrote debugging wiki. : Completed - Test PRs - tested PR4661: Completed

Anton:
- Create Wiki pages for VR and saucelabs connection – completed
- Write e2e tests Inline help component (Core) – completed
- Write e2e tests Info label component (Core) – completed
- Applitools PoC (visual regression refactoring) - in progress - refactor hardcoded colors - not started - checkbox e2e tests - not started - write test cases for inline help - not started - reviewed PRs

Nikita:
-Add feed input e2e tests core : Completed
- Address PR comments - 7 PRs updated and being re-reviewed - Text e2e tests - not started - List by Line e2e tests - not started - raised issues - reviewed PRs

Puru: Completed Testing Avatar
Completed Manual test cases for Tree component
Working for Defect report
Testing PRs (2233, 5064, 5067, 5066)
Defect Hunting testing


Number of components targetted:7
Number of components completed: 1 (form facets)
Number of tech specs targetted: 3
Number of tech specs presented:1 (Rich Text Editor-final approval pending)
